lv_misc/lv_log.h

```diff
@@ -64,10 +64,26 @@ void lv_log_add(lv_log_level_t level, const char * file, int line, const char *
  *      MACROS
  **********************/
 
-#define LV_LOG_TRACE(dsc)   lv_log_add(LV_LOG_LEVEL_TRACE, __FILE__, __LINE__, dsc);
-#define LV_LOG_INFO(dsc)    lv_log_add(LV_LOG_LEVEL_INFO, __FILE__, __LINE__, dsc);
-#define LV_LOG_WARN(dsc)    lv_log_add(LV_LOG_LEVEL_WARN, __FILE__, __LINE__, dsc);
-#define LV_LOG_ERROR(dsc)   lv_log_add(LV_LOG_LEVEL_ERROR, __FILE__, __LINE__, dsc);
+#if LV_LOG_LEVEL <= LV_LOG_LEVEL_TRACE
+#  define LV_LOG_TRACE(dsc)   lv_log_add(LV_LOG_LEVEL_TRACE, __FILE__, __LINE__, dsc);
+#else
+#  define LV_LOG_TRACE(dsc)   {;}
```

Comment:
Usually I use `do {} while(0)` here; are you sure this has the same effect?

Reply:
This is used later in the same file.

Reply:
@averne Fair enough.